aws/apigateway_test.go

```diff
@@ -130,7 +132,21 @@ func TestNukeAPIGatewayMoreThanOne(t *testing.T) {
 	region, err := getRandomRegion()
 	require.NoError(t, err)
 
-	session, err := session.NewSession(&aws.Config{Region: aws.String(region)})
+	session, err := session.NewSession(
+		&aws.Config{
+			Region: aws.String(region),
+
+			// The defaultretryer is overridden here as the default values for MinRetryDelay and MinThrottleDelay
+			// are too low for the API Gateway API.
+			// https://pkg.go.dev/github.com/aws/aws-sdk-go@v1.44.274/aws/client#DefaultRetryer
+			Retryer: client.DefaultRetryer{
```

Comment:
I'm not sure if you'd want this here, but offering the ability to increase the min retry on the session as a way to get over the rate limit exceptions on the API Gateway createrestapi in the tests.

If useful it would be better as a generic test function, that all tests can reference.

Also happy to back this out if you don't want to adjust the session config.

Reply:
Thanks for bringing this up. Can you create a new issue to introduce retry logic similar to this?  I think it would be better to revert this change for this PR, as it's irrelevant to ACM support.